libs/sdk-core/src/breez_services.rs

```diff
@@ -879,7 +879,11 @@ impl BreezServices {
         self.connect_lsp_peer(node_pubkey).await?;
 
         // First query the changes since last sync time.
-        let since_timestamp = self.persister.last_payment_timestamp().unwrap_or(0);
+        let since_timestamp = self
+            .persister
+            .get_last_sync_time()?
+            .or_else(|| self.persister.last_payment_timestamp().ok())
```

Comment:
Do we need the fallback to `last_payment_timestamp` ?

Reply:
The fallback is for the first sync that happens before the `last_sync_time` is set. We could alternatively set it when building the BreezService instance. What do you think?

Reply:
I see there is already fallback to zero, that's why I was asking.

Reply:
Yeah, it could be the first run/node without payments, so also needs a fallback if there is no payment also. Any recommendations?

Reply:
How about leaving the zero as single fallback? Worst case on migration it will sync all payments one time.

Reply:
I'm fine with that. First sync after update will definely be a full payment sync.